website/content/en/status/report-2023-01-2023-03/clusteradm.adoc

```diff
@@ -43,7 +43,7 @@ The hardware and network connection have been generously provided by:
 * https://www.isc.org/[Internet Systems Consortium]
 * https://www.inx.net.za/[INX-ZA]
 * https://www.kddi-webcommunications.co.jp/[KDDI Web Communications Inc]
-* https://myren.net.my/[Malaysian Research & Education Network]
+* https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
```

Comment:
Also need to update the [internal/about](https://www.freebsd.org/internal/about/) page.
Could you take care of it, please? Along with the other URLs?

Then I will update the same in the mirrors:
http://pkg0.kul.freebsd.org/
http://ftp0.kul.freebsd.org/

Reply:
Thanks

> Also need to update the [internal/about](https://www.freebsd.org/internal/about/) page. Could you take care of it, please? Along with the other URLs?

▶ https://github.com/freebsd/freebsd-doc/pull/157

> Then I will update the same in the mirrors: [http://pkg0.kul.freebsd.org](http://pkg0.kul.freebsd.org/) [http://ftp0.kul.freebsd.org](http://ftp0.kul.freebsd.org/)